### DISCUSSION TITLE
我部署下来为啥使用插件DALL E3 之后，不出图片，一直在转圈，是需要啥东西吗？

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1226] [General Question]
我部署下来为啥使用插件DALL E3 之后，不出图片，一直在转圈，是需要啥东西吗？

**Comment by arvinxx** [comment]
看下请求？可能是你有 api 限制？

**Comment by 5478-lfy** [comment]
我用的是Azure OpenAI的API


函数调用
{
  "apiName": "text2image",
  "arguments": "{\n  \"prompts\": [\n    \"Photo of a handsome duck with glossy feathers reflecting sunlight, standing proudly on a clear blue lake with ripples around it.\",\n    \"Illustration of a stylish duck wearing a bow tie and sunglasses, posing with a confident smirk on a sunny park bench.\",\n    \"Cartoon of an adorable duck with a sleek feather hairstyle, flexing its wings in front of a mirror in a lush green meadow.\",\n    \"Watercolor painting of an elegant duck with a striking plumage pattern, wading gracefully in a serene pond surrounded by water lilies.\"\n  ],\n  \"quality\": \"hd\"\n}",
  "identifier": "lobe-image-designer",
  "type": "builtin"
}
调用参数
{
  "prompts": [
    "Photo of a handsome duck with glossy feathers reflecting sunlight, standing proudly on a clear blue lake with ripples around it.",
    "Illustration of a stylish duck wearing a bow tie and sunglasses, posing with a confident smirk on a sunny park bench.",
    "Cartoon of an adorable duck with a sleek feather hairstyle, flexing its wings in front of a mirror in a lush green meadow.",
    "Watercolor painting of an elegant duck with a striking plumage pattern, wading gracefully in a serene pond surrounded by water lilies."
  ],
  "quality": "hd"
}
返回结果
[
  {
    "prompt": "Photo of a handsome duck with glossy feathers reflecting sunlight, standing proudly on a clear blue lake with ripples around it.",
    "quality": "hd",
    "size": "1024x1024",
    "style": "vivid"
  },
  {
    "prompt": "Illustration of a stylish duck wearing a bow tie and sunglasses, posing with a confident smirk on a sunny park bench.",
    "quality": "hd",
    "size": "1024x1024",
    "style": "vivid"
  },
  {
    "prompt": "Cartoon of an adorable duck with a sleek feather hairstyle, flexing its wings in front of a mirror in a lush green meadow.",
    "quality": "hd",
    "size": "1024x1024",
    "style": "vivid"
  },
  {
    "prompt": "Watercolor painting of an elegant duck with a striking plumage pattern, wading gracefully in a serene pond surrounded by water lilies.",
    "quality": "hd",
    "size": "1024x1024",
    "style": "vivid"
  }
]

**Reply by 5478-lfy** [reply]
@arvinxx

**Comment by MazzaWill** [comment]
我也有同样的问题，一直在转圈，看不到生成的图片

**Reply by arvinxx** [reply]
看下请求，大概率是被限制了…
